ansible/vars/dev.yml

```diff
@@ -9,7 +9,7 @@ ECRYPTFS_PASSWORD: 'xxBvYiMZ3YVIVkFR'
 KSPLICE_ACTIVE: no
 KSPLICE_ACTIVATION_KEY:
 
-AMQP_USER: hq_worker
+AMQP_USER: "{{ deploy_env }}_worker"
```

Comment:
On staging its `staging_worker`, on prod its `hq_worker`.  I had only checked staging.  I believe that a full `ansible-playbook deploy_stack.yml ...` call would appropriately switch the worker name though.  It would add the user in rabbitmq, then update the localsettings to point at that new user.